assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -1,75 +1,99 @@
-import React from "react";
+import React, { useContext, useState } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
+import { WorkFormContext } from "../FormProvider";
+
+const visibilityOptions = ["RESTRICTED", "WHAT", "ARE", "THESE"];
```

Comment:
These are an enum under the work type object. We could expose a way to get the list and terms through a query. For now you could probably hardcode the options.
<img width="632" alt="Screen Shot 2020-02-26 at 9 06 45 AM" src="https://user-images.githubusercontent.com/6372022/75357710-6cec4f00-5877-11ea-8236-203c54cca5be.png">

Reply:
@kdid Got it.  I was kinda leaving those in for @katdivyareddy10 to find in Graphiql API.  I figured they'd be defined someplace, so good to know.